testprojects/src/python/python_distribution/ctypes/BUILD

```diff
@@ -31,4 +31,5 @@ python_binary(
   dependencies=[
     ':ctypes',
   ],
+  platforms="current"
```

Comment:
It may be true `platforms` accepts string or list of strings, but its plural and this feature is not documented. Stick to `['current']` for local readability? I had to go check all this before flagging as a bug for example since its not locally readable.